frontend/src/main/scala/bloop/exec/Forker.scala

```diff
@@ -200,6 +200,11 @@ object Forker {
       val envMap = builder.environment()
       envMap.putAll(env.asJava)
       builder.redirectErrorStream(false)
+      logger.info("Forking process:")
```

Comment:
Would we want it under debug rather?

Reply:
How can we get debug logs with Bloop?

Reply:
we would need to add a flag to starting params. Maybe it's fine to do info.